lib/web_ui/test/channel_buffers_test.dart

```diff
@@ -40,30 +38,30 @@ void testMain() {
     final ByteData data = _makeByteData('bar');
     final ui.ChannelBuffers buffers = ui.ChannelBuffers();
     bool called = false;
-    final ui.PlatformMessageResponseCallback callback = (ByteData responseData) {
+    final ui.PlatformMessageResponseCallback callback = (ByteData? responseData) {
       called = true;
     };
     buffers.push(channel, data, callback);
-    await buffers.drain(channel, (ByteData drainedData, ui.PlatformMessageResponseCallback drainedCallback) {
+    await buffers.drain(channel, (ByteData? drainedData, ui.PlatformMessageResponseCallback drainedCallback) {
       expect(drainedData, equals(data));
       assert(!called);
       drainedCallback(drainedData);
       assert(called);
-      return;
+      return Future<void>.value();
```

Comment:
Adding `async` runs the body of the function in a microtask, so adding it would alter the semantic of the test. I didn't want to do that yet as this test is supposed to be nearly identical to a similar test on the mobile side: https://github.com/flutter/engine/blob/master/testing/dart/channel_buffers_test.dart#L7